src/app/components/shell/microsurvey-popup/adoption-content.js

```diff
@@ -60,6 +60,9 @@ function useDismissalCookie() {
 function AdoptionContentBase({children, disable}) {
     const {userModel} = useUserContext();
     const {first_name: name} = userModel || {};
+    const {pathname} = useLocation();
+    const href = `${window.location.origin}${pathname}`;
+    const renewalFormHref = `/renewal-form?from=popup&href=${encodeURIComponent(href)}`;
```

Comment:
the meaning of the `href` param isn't really clear to me, would `referrer` be more correct? or `from_url`?

Reply:
I have changed it to `returnTo`, as it is the url to return to after submitting the form.